bindings/rust/wasmedge-macro/README.md

```
@@ -1,6 +1,6 @@
 # Overview
 
-** `wasmedge-mcaro` is under [wasmedge-rust-sdk/crates/wasmedge-mcaro](https://github.com/WasmEdge/wasmedge-rust-sdk/tree/main/crates/wasmedge-mcaro) now. **
+** `wasmedge-macro` is under [wasmedge-rust-sdk/crates/wasmedge-macro](https://github.com/WasmEdge/wasmedge-rust-sdk/tree/main/crates/wasmedge-macro) now. **
```

markdownlint-cli2 (0.20.0)

Spaces inside emphasis markers

(MD037, no-space-in-emphasis)